frontend/src/app/application-forms/tree-application-form/tree-permit-view/tree-permit-view.component.html

```diff
@@ -66,8 +76,23 @@ <h2 class="usa-heading">Order details</h2>
   <div id="tree-additional-info-container" class="usa-grid confirmation-details-container margin-top-2 margin-bottom-2">
     <h2 class="usa-heading">Additional information</h2>
     <hr class="header-divider">
-    <b>Your permit will be emailed to you at {{permit.emailAddress}}.</b>
-    <p *ngIf="forest">Return to the <a [href]="'/christmas-trees/forests/' + forest.forestAbbr + '/tree-guidelines'">{{forest.forestName}} National Forest{{ forest.forestName.indexOf('and') > 0 ? 's' : ''}} Christmas tree permit website.</a></p>
+    <h3>Your permit was emailed to you at {{permit.emailAddress}}.</h3>
+    <p *ngIf="forest">Return to the <a [href]="'/christmas-trees/forests/' + forest.forestAbbr + '/tree-guidelines'">{{forest.forestName}} National Forest{{ forest.forestName.indexOf(' and ') > 0 ? 's' : ''}} Christmas tree permit website.</a></p>
   </div>
 
 </section>
+
+<section *ngIf="error && permit && forest" id="application-view" class="usa-section">
+  <div class="usa-grid">
+    <div id="pay-gov-errors" class="usa-alert usa-alert-error" aria-live="assertive" aria-hidden="false" role="alert">
+      <div class="usa-alert-body">
+        <h3 class="usa-alert-heading">Unable to process your permit. Your credit card has not been charged.</h3>
+        <p class="usa-alert-text">{{error.message}}</p>
+      </div>
+    </div>
+    <div class="margin-top-2 margin-bottom-2">
+      <p *ngIf="forest"><strong>Return to the <a [href]="'/applications/christmas-trees/forests/' + forest.forestAbbr + '/new/' + permit.permitId">{{forest.forestName}} National Forest{{ forest.forestName.indexOf(' and ') > 0 ? 's' : ''}} Christmas tree permit purchase form</a> and try again.</strong></p>
+      <p>If the problem persists, please contact the U.S. Forest Service at (800) 832-1355 or <a href="fs-permitting-project@gsa.gov">fs-permitting-project@gsa.gov.</a></p>
```

Comment:
the href should probably be a `mailto:` unless that has accessibility implications.

Reply:
updated to include mailto in sprint-7 story # 285